drivers/scale_service.go

```diff
@@ -66,6 +66,9 @@ func (s *ScaleServiceDriver) Execute(conf interface{}, apiClient client.RancherC
 	if scaleAction == "up" {
 		newScale = service.Scale + int64(scaleChange)
 	} else if scaleAction == "down" {
+		if service.Scale-int64(scaleChange) < 0 {
```

Comment:
This will change when you implement min/max fields, but can you change this to <= 0 for now?